pkg/services/accesscontrol/cacheutils.go

```diff
@@ -41,3 +43,12 @@ func GetBasicRolePermissionCacheKey(role string, orgID int64) string {
 func GetTeamPermissionCacheKey(teamID int64, orgID int64) string {
 	return fmt.Sprintf("rbac-permissions-team-%d-%d", orgID, teamID)
 }
+
+func GetTeamPermissionCompositeCacheKey(teamIds []int64, orgID int64) string {
+	teams := make([]string, 0)
+	slices.Sort(teamIds)
```

Comment:
Should the sort be on the `teams` slice after the append loop? In the case now, you'd be sorting the slice passed into this function which seems like an unintended side-effect.

Reply:
Yeah, that's true, since Sort() does not return new slice, but modifies original one.